docs/ui/components/Sidebar/SidebarHeadEntry.tsx

```diff
@@ -14,14 +14,14 @@ type SidebarHeadEntryProps = {
 export const SidebarHeadEntry = ({ href, title, isActive, Icon }: SidebarHeadEntryProps) => {
   return (
     <Link href={href} passHref>
-      <div css={[entryContainerStyle, isActive && activeEntryContainerStyle]}>
+      <a css={[entryContainerStyle, isActive && activeEntryContainerStyle]}>
         <Icon
```

Comment:
This icon probably also should not be announced. This leads to:

```text
Plan-enterprise icon Expo Application Services visited link
```

If `aria-hidden="true"` is added to the icon it becomes:

```
Expo Application Services visited link
```